kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/ListReassignmentsResponse.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
I believe this file is left-over from the merge. This got renamed ListAllReassignmentsResponse. Please, delete.

Reply:
removed.